engine/installation/linux/oracle.md

```diff
@@ -16,27 +16,36 @@ To get started with Docker on Oracle Linux, make sure you
 
 To install Docker, you need the 64-bit version of Oracle Linux 6 or 7.
 
-To use `btrfs`, you need to install the Unbreakable Enterprise Kernel (UEK)
+Oracle recommends using the Unbreakable Enterprise Kernel (UEK)
```

Comment:
Please remove "Oracle recommends". Maybe you can quantify why the UEK kernel is recommended instead.